environment.yml

```diff
@@ -24,6 +24,7 @@ dependencies:
   - isort  # check that imports are in the right order
   - mypy
   - pycodestyle  # used by flake8
+  - typing_extensions
```

Comment:
can you add a note in 1.0 docs (may have to add a new section on dependencies like we did in 0.25) and list this